tests/PHPStan/Analyser/data/cli-globals.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace CliGlobals;
+
+\PHPStan\Testing\assertType('int<1, max>', $argc);
+\PHPStan\Testing\assertType('non-empty-array<int, string>', $argv);
```

Comment:
$argc/$argv always count/contain the program name you pass `php test.php`, so actually you never have a empty type here.